### PR TITLE
Fix some of the log messages from tv.py

### PR DESCRIPTION
**Pull request by fernandog**
@OmgImAlexis i think this will help you

@miigotu

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: sickbeard/tv.py [hunk 1]
```diff
@@ -500,25 +501,26 @@ def loadEpisodesFromDB(self):
...
         for curResult in sqlResults:
 
-            logger.log(u"loadEpisodesFromDB curResult: " + str(curResult), logger.DEBUG)
-            deleteEp = False
-
             curSeason = int(curResult["season"])
             curEpisode = int(curResult["episode"])
+            curShowid = int(curResult['showid'])
+
+            logger.log(u"%s: loading Episodes from DB" % curShowid, logger.DEBUG)
```
Comment (miigotu) [inline sickbeard/tv.py]: spam, this is inside a loop. and you removed the curResult

Comment (fernandog) [inline sickbeard/tv.py]: @miigotu this was returning empty in log: str(curResult)

I created curShowid = int(curResult['showid'])

Or show do I show all items in curResult?

Comment (miigotu) [inline sickbeard/tv.py]: ugh ok, I read that wrong.

**Pull request merged**